### PR TITLE
my resturant data completed. upload photo completed.

### PR DESCRIPTION
**Pull request by giacomofrigo**

### CODE REVIEW

**Comment by francescofact** [comment]
for me ok, waiting for @vincenzopalazzo rebase

**Review by francescofact — APPROVED** [review]

**Comment by vincenzopalazzo** [comment]
I will rebase this PR and I will manage the propriety name, sorry about that

**Review by vincenzopalazzo — APPROVED** [review]
@vincenzopalazzo approved this PR and also Travis. There is some coverage decrease but it is totally normal.

File: monolith/views/restaurants.py [hunk 1]
```diff
@@ -211,37 +222,95 @@ def my_reservations():
...
+            photo_gallery = PhotoGallery()
+            photo_gallery.caption = form.data["caption"]
+            photo_gallery.url = form.data["url"]
+            photo_gallery.restaurant_id = session["RESTAURANT_ID"]
+            db.session.add(photo_gallery)
+            db.session.commit()
+
+        return redirect("/my_restaurant_photogallery")
+    else:
+        photos = PhotoGallery.query.filter_by(
```
Comment (vincenzopalazzo) [inline monolith/views/restaurants.py:307]: If I'm on the page of the restaurant, I can click on the photo gallery and with a get request I will go inside this method but, for some reason, I don't see where you store the `RESTAURANT_ID`.

I receive this exception

```
Traceback (most recent call last):
  File "/home/vincent/.local/lib/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/vincent/.local/lib/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/home/vincent/.local/lib/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/vincent/.local/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/vincent/.local/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/vincent/.local/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/vincent/.local/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/vincent/.local/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/vincent/.local/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/vincent/.local/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/vincent/.local/lib/python3.6/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/home/vincent/GithubUnipi/GoOutSafe_Primer2020/monolith/auth.py", line 35, in f
    return func(*args, **kwargs)
  File "/home/vincent/GithubUnipi/GoOutSafe_Primer2020/monolith/views/restaurants.py", line 308, in my_photogallery
    restaurant_id=session["RESTAURANT_ID"]
  File "/usr/local/lib/python3.6/dist-packages/werkzeug/local.py", line 377, in <lambda>
    __getitem__ = lambda x, i: x._get_current_object()[i]
  File "/home/vincent/.local/lib/python3.6/site-packages/flask/sessions.py", line 84, in __getitem__
    return super(SecureCookieSession, self).__getitem__(key)
KeyError: 'RESTAURANT_ID'
127.0.0.1 - - [27/Oct/2020 21:59:46] "GET /my_restaurant_photogallery?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
127.0.0.1 - - [27/Oct/2020 21:59:46] "GET /my_restaurant_photogallery?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
127.0.0.1 - - [27/Oct/2020 21:59:46] "GET /my_restaurant_photogallery?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
127.0.0.1 - - [27/Oct/2020 21:59:46] "GET /my_restaurant_photogallery?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -

```

**Review by vincenzopalazzo — COMMENTED** [review]
I have a very small comment. If you want help with this problem you can convert it inside the issue. But maybe for you is easy to find the problem inside your changes

File: monolith/views/restaurants.py (hunk 1 above)
Comment (vincenzopalazzo) [reply monolith/views/restaurants.py:309]: Fixed for now and tested it

**Comment by vincenzopalazzo** [comment]
Restored also coverage of code! Thanks travis to give me the possibility to merge the PR

**Pull request merged**